backend/controller/personalpurchases.controller.js

```diff
@@ -28,11 +31,35 @@ const createPersonalPurchaseController = (req, res) => {
 }
 
 const updatePersonalPurchaseController = (req, res) => {
+    if (req.body.fi_link) {
+        res.status(400).json('Error: fi_link in PPR cannot be patched')
```

Comment:
"cannot be patched" is a little misleading, its more cannot be patched via this endpoint